config/locales/en.yml

```diff
@@ -234,9 +234,14 @@ en:
     new:
       title: Create a new service
       heading: Add a service
+    edit:
+      description: Edit service
```

Comment:
it's primarily used as a page title
```suggestion
      title: Edit service
```

Reply:
It is used in the title and heading

Reply:
so why would it be called `description` if it's not used in a description at all?